jenkins/ci.suse.de/openstack-submit-project.yaml

```diff
@@ -25,90 +25,91 @@
 
     builders:
       - shell: |
-          # DEBUG MODE with E=echo
-          # NORMAL MODE with E=
+          # to debug set E=echo
           E=
 
-          if [ -z "$starttime" ] ; then
+          if [[ ! $starttime ]] ; then
               echo "FATAL: calling script needs to specify starttime"
               exit 1
           fi
 
           case $subproject in
-          None) echo "No valid 'subproject' provided - defaults for safety reasons to 'None'"
-                exit 1
-          ;;
-          "-")  echo "Subproject set to '-': skip job, do not fail (eg. for Master project)"
-                exit 0
-          ;;
-          esac
-
-          export BS=IBS
-          case $BS in
-          OBS)
-              OSC="osc -A https://api.opensuse.org"
-          ;;
-          IBS)
-              OSC="osc -A https://api.suse.de"
-          ;;
+              None) echo "No valid 'subproject' provided - defaults for safety reasons to 'None'"
+                    exit 1
+              ;;
+              "-")  echo "Subproject set to '-': skip job, do not fail (eg. for Master project)"
+                    exit 0
+              ;;
           esac
 
+          export bs=IBS
+          osc_obs="osc -A https://api.opensuse.org"
```

Comment:
2 unused variables?

Reply:
`bs` is used

Reply:
and the osc_obs variable was used in earlier versions of the script (when the job still was a stand alone shell script)
sure it can be dropped